### ISSUE TITLE
System Error when running Ganache 2.0.1 on darwin again

### ISSUE THREAD

**Issue by FlashSheridan** [trufflesuite/ganache#1471]
Similar to https://github.com/trufflesuite/ganache/issues/1448, but this time from Visual Studio Code.

PLATFORM: darwin
GANACHE VERSION: 2.0.1

EXCEPTION:
```
TypeError: Cannot read property 'pop' of undefined
    at CheckpointTrie.Trie._updateNode (/node_modules/ganache-core/node_modules/merkle-patricia-tree/baseTrie.js:360:23)
    at /node_modules/ganache-core/node_modules/merkle-patricia-tree/baseTrie.js:107:16
    at /node_modules/ganache-core/node_modules/merkle-patricia-tree/baseTrie.js:461:14
    at processNode (/node_modules/ganache-core/node_modules/merkle-patricia-tree/baseTrie.js:471:23)
    at /node_modules/ganache-core/node_modules/merkle-patricia-tree/baseTrie.js:516:13
    at /node_modules/ganache-core/node_modules/merkle-patricia-tree/baseTrie.js:180:7
    at /node_modules/ganache-core/node_modules/merkle-patricia-tree/util.js:75:7
    at /node_modules/ganache-core/node_modules/merkle-patricia-tree/node_modules/async/lib/async.js:52:16
    at /node_modules/ganache-core/node_modules/merkle-patricia-tree/node_modules/async/lib/async.js:269:32
    at /node_modules/ganache-core/node_modules/merkle-patricia-tree/node_modules/async/lib/async.js:44:16
    at /node_modules/ganache-core/node_modules/merkle-patricia-tree/util.js:71:7
    at /node_modules/ganache-core/node_modules/merkle-patricia-tree/baseTrie.js:157:9
    at /node_modules/ganache-core/lib/database/levelupobjectadapter.js:41:16
    at /node_modules/ganache-core/node_modules/level-sublevel/shell.js:101:15
    at /node_modules/ganache-core/node_modules/level-sublevel/nut.js:121:19
    at /node_modules/ganache-core/node_modules/encoding-down/index.js:51:21
    at /node_modules/ganache-core/node_modules/cachedown/index.js:58:21
    at ReadFileContext.callback (/node_modules/ganache-core/lib/database/filedown.js:26:14)
    at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:420:13)
```

APPLICATION LOG:
```
T+536706644ms:   Block Number: 831
T+536706644ms:   Block Time: Mon Nov 04 2019 19:46:52 GMT-0500 (EST)
T+536706644ms: eth_getBlockByNumber
T+536706644ms: eth_getTransactionReceipt
T+536707194ms: web3_clientVersion
T+536707194ms: eth_accounts
T+536707195ms: eth_call
T+536707750ms: web3_clientVersion
T+536707755ms: eth_accounts
T+536707794ms: eth_sendTransaction
T+536707794ms:   Transaction: 0x3891b63c18b3b844c97225710d0618045579219c93384eeb3f4f961e5a9e9fd7
T+536707794ms:   Contract created: 0xd5ed2df73c114f2d98e83f90956d01090779a748
T+536707794ms:   Gas usage: 84562
T+536707794ms:   Block Number: 832
T+536707794ms:   Block Time: Mon Nov 04 2019 19:46:54 GMT-0500 (EST)
T+536707794ms: eth_getBlockByNumber
T+536707796ms: eth_getTransactionReceipt
T+536708344ms: web3_clientVersion
T+536708344ms: eth_accounts
T+536708358ms: eth_call
T+536714306ms: web3_clientVersion
T+536714344ms: eth_accounts
T+536714344ms: eth_sendTransaction
T+536714360ms:   Transaction: 0x31ffc6a068d92cb37a2d2fdfe3dfb8aafcb8bd0ef050a774268528c987269578
T+536714360ms:   Contract created: 0x9e6f17500e8ad21d996d99db10a9fb7e76374ddb
T+536714360ms:   Gas usage: 370820
T+536714360ms:   Block Number: 833
T+536714360ms:   Block Time: Mon Nov 04 2019 19:47:00 GMT-0500 (EST)
T+536714394ms: eth_getBlockByNumber
T+536714394ms: eth_getTransactionReceipt
T+536714944ms: web3_clientVersion
T+536714944ms: eth_accounts
T+536714959ms: eth_sendTransaction
T+536715044ms:   Transaction: 0x974fcfec5b24f765b92c0d43e5b1e066900e33309ffe95f6fc0ef512908dce54
T+536715044ms:   Contract created: 0x17880895e325b95e9df6b15da819929e8eced317
T+536715044ms:   Gas usage: 370820
T+536715044ms:   Block Number: 834
T+536715044ms:   Block Time: Mon Nov 04 2019 19:47:01 GMT-0500 (EST)
T+536715044ms: eth_getBlockByNumber
T+536715044ms: eth_getTransactionReceipt
T+536715594ms: web3_clientVersion
T+536715594ms: eth_accounts
T+536715644ms: eth_sendTransaction
T+536715644ms:   Transaction: 0xf66fb1002b3bce8fe9e39a48b614102fac1bbfdd5a96cdbdbb27368f379e2960
T+536715644ms:   Gas usage: 23205
T+536715644ms:   Block Number: 835
T+536715644ms:   Block Time: Mon Nov 04 2019 19:47:01 GMT-0500 (EST)
T+536715644ms: eth_getBlockByNumber
T+536715644ms: eth_getTransactionReceipt
T+536716246ms: web3_clientVersion
T+536716252ms: eth_accounts
T+764911999ms: web3_clientVersion
T+764912052ms: eth_accounts
T+764912099ms: eth_accounts
T+764912099ms: eth_accounts
T+764912102ms: eth_accounts
T+764912113ms: eth_accounts
T+764912150ms: eth_accounts
T+764912159ms: eth_accounts
T+764912197ms: eth_accounts
T+764912197ms: eth_accounts
T+764912200ms: eth_accounts
T+764912216ms: eth_accounts
T+764912248ms: eth_accounts
T+764912248ms: eth_accounts
T+764912258ms: eth_accounts
T+764912297ms: eth_accounts
T+764912297ms: eth_accounts
T+764912300ms: eth_accounts
T+764912312ms: eth_accounts
T+764912348ms: eth_accounts
T+764912348ms: eth_accounts
T+764912358ms: eth_accounts
T+764912396ms: eth_accounts
T+764912396ms: eth_accounts
T+764912396ms: eth_accounts
T+764912406ms: eth_accounts
T+764912447ms: eth_accounts
T+764912447ms: eth_accounts
T+764912447ms: eth_accounts
T+764912459ms: eth_accounts
T+764912496ms: eth_accounts
T+764912496ms: eth_accounts
T+764912506ms: eth_accounts
T+764912545ms: eth_accounts
T+764912545ms: eth_accounts
T+764912545ms: eth_accounts
T+764912561ms: eth_accounts
T+764912597ms: eth_accounts
T+764912597ms: eth_accounts
T+764912611ms: eth_accounts
T+764912647ms: eth_accounts
T+764912647ms: eth_accounts
T+764912696ms: eth_accounts
T+764912696ms: eth_accounts
T+764912708ms: eth_accounts
T+764912752ms: eth_accounts
T+764912752ms: eth_accounts
T+764913948ms: eth_sendTransaction
T+764915205ms: eth_unsubscribe
T+764915205ms: eth_unsubscribe
T+764915360ms: eth_unsubscribe
T+764915360ms: eth_unsubscribe
```

**Comment by nicholasjpaterno** [comment]
@FlashSheridan Thanks for opening this issue.  It looks like you're using an outdated version of Ganache UI.  I believe this issue has been fixed.  Try https://github.com/trufflesuite/ganache/releases/latest and let us know if this issue persists!

I'm going to close this for now.  Please reopen if this isn't resolved!

**Comment by wbt** [comment]
It has not been resolved, but I think it's a duplicate of [#1320](https://github.com/trufflesuite/ganache/issues/1320).
